heron/ui/src/python/main.py

```diff
@@ -134,14 +136,15 @@ def main():
 
   # log additional information
   command_line_args = vars(parsed_args)
-  address = socket.gethostbyname(socket.gethostname())
+
+  address = '0.0.0.0' if command_line_args['address'] is None else command_line_args['address']
```

Comment:
@kramasamy sure,
127.0.0.1      	    localhost
255.255.255.255	broadcasthost
::1             localhost

May I know LISTEN when you run heron-ui too?
$ lsof -i -P | grep LISTEN